### ISSUE TITLE
Module not found if 'use lib' contains an environment variable

### ISSUE THREAD

**Issue by landona** [Camelcade/Perl5-IDEA#2412]
A module is not found if it depends on a `use lib "<path>"` and path contains an environment variable, e.g.
`use lib $ENV{PROJECT_ROOT}/lib;`
`use MyModule;`  

* IntelliJ IDEA Community Ed. 2021.1.3, #IC-211.7628.21 
* Perl5 plugin 2021.1.2
* OpenJDK version "1.8.0_262"
* Red Hat Enterprise Linux 7

**Comment by hurricup** [comment]
Yes, this is not supported atm. You may just mark this directory as library sources manually. 
Not sure if this will ever work.

**Comment by landona** [comment]
Yes, already did.
Thanks for the feedback!
